src/prefect/deployments/runner.py

```diff
@@ -767,28 +783,61 @@ def local_flow():
 
         console.print(f"Successfully pushed image {image.reference!r}", style="green")
 
+    deployment_exceptions = []
     deployment_ids = []
     for deployment in track(
-        args,
+        deployments,
         description="Creating/updating deployments...",
         console=console,
+        transient=True,
     ):
-        deployment_ids.append(
-            await deployment.apply(image=image.reference, work_pool_name=work_pool_name)
-        )
+        try:
+            deployment_ids.append(
+                await deployment.apply(
+                    image=image.reference, work_pool_name=work_pool_name
+                )
+            )
+        except Exception as exc:
+            if len(deployments) == 1:
+                raise
+            deployment_exceptions.append({"deployment": deployment, "exc": exc})
 
-    console.print("Successfully created/updated all deployments!\n", style="green")
+    if deployment_exceptions:
+        console.print(
+            "Encountered errors while creating/updating deployments\n",
+            style="orange_red1",
+        )
```

Comment:
```suggestion
    if deployment_exceptions:
        console.print(
            "Encountered errors while creating/updating deployments\n",
            style="orange_red1",
        )
```
I think a colon after this would look nice, based on the screenshot.